platform/android/MapboxGLAndroidSDK/src/main/res/values/dimens.xml

```diff
@@ -6,4 +6,13 @@
     <dimen name="mapbox_eight_dp">8dp</dimen>
     <dimen name="mapbox_ninety_two_dp">92dp</dimen>
     <dimen name="mapbox_my_locationview_outer_circle">18dp</dimen>
+
+    <!--Minimum scale velocity required to start animation-->
+    <dimen name="mapbox_minimum_scale_velocity">150dp</dimen>
+
+    <!--Minimum scale span delta required to execute scale gesture when rotating-->
+    <dimen name="mapbox_minimum_scale_span_when_rotating">100dp</dimen>
+
+    <!--Minimum angular velocity required to start rotation animation-->
+    <dimen name="mapbox_minimum_angular_velocity">0.025dp</dimen>
```

Comment:
are above public resources (is a user suppose to override these?)? if yes, these should be exposed  in public.xml

Reply:
Following up on this in https://github.com/mapbox/mapbox-gl-native/issues/11331.